packages/shopify-api/docs/guides/session-storage.md

```diff
@@ -65,6 +65,8 @@ Now that the app has a JavaScript object containing the data of a `Session`, it
 
 The `Session` class also includes an instance method called `.toPropertyArray` that returns an array of key-value pairs, e.g.,
 
+`toPropertyArray` has an optional parameter `returnUserData`, defaulted to false, when set to true it will return the associated user data as part of the property array object.
```

Comment:
We should remember to update this to say this will be the way in the future when we do the deprecation.